packages/surveys/src/types.ts

```diff
@@ -0,0 +1,35 @@
+/* Copyright (c) 2020 Environmental Systems Research Institute, Inc.
+ * Apache-2.0 */
+
+import { IRequestOptions } from "@esri/arcgis-rest-request";
+import { IItem } from "@esri/arcgis-rest-types";
+
+type IResultsAvailability = "after" | "always";
+
+export interface IFormItemProperties {
+  [propName: string]: any;
+  previewUrl?: string;
+  parentId?: string;
+  settings?: {
+    resultsAvailability: IResultsAvailability;
+  };
+}
+
+export interface IFormItem extends IItem {
+  id: string;
```

Comment:
We get this prop from [IItem](https://github.com/Esri/arcgis-rest-js/blob/v2.13.0/packages/arcgis-rest-types/src/item.ts#L57)